README.md

```diff
@@ -135,6 +141,28 @@ This can be done either by using the internal daemon or cron.
    screen -S miflora-mqtt-daemon -d -m python3 /path/to/miflora-mqtt-daemon.py
    ```
 
+## Usage with Docker
+
+A Dockerfile in the repository can be used to build a docker container from the sources with a command such as:
+
+```shell
+docker build -t miflora-mqtt-daemon .
+```
+
+Running the container in interactive mode works like this:
+
+```shell
+docker run -it --name miflora-mqtt-daemon -v /opt/config/:/config miflora-mqtt-daemon
+```
+
+To run the container in daemon mode use `-d` flag:
+
+```shell
+docker run -d --name miflora-mqtt-daemon -v /opt/config/:/config miflora-mqtt-daemon
```

Comment:
I was talking about this line.

Chances are high a user executing the command will execute it in the cloned miflora folder, wherever it might be. Hence it would be obvious to mount: `-v .:/config`